modules/candidate_list/php/candidate_list.class.inc

```diff
@@ -152,8 +152,6 @@ class Candidate_List extends \NDB_Menu_Filter
                                'participantstatus' => $participant_status_options,
                                'useedc'            => $config->getSetting("useEDC"),
                               ];
-
-        return true;
```

Comment:
When you see this line deleted in this PR in other files, it typically means that the function should actually return void.